_script/deploy

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 set -exuo pipefail
-
 cd $(dirname $0)/..
 
-docker push dachad/tcpgoon
+docker_name=${1:-"dachad/tcpgoon"}
```

Comment:
?

Reply:
I assume that it's a default initialisation of the variable in case of no value in the variable, am I right?

Reply:
you are right! Why do you comment on outdated commits? DO you check every single version of the commited files or the files history?

Reply:
Yes, I go commit by commit from top-down to understand the history of changes... I think it's a good practice, but it's a bit difficult to follow you :trollface: